QUEST.tsv

```diff
@@ -2765,252 +2765,252 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
+QUEST_20151022_002832	Since you carry such malice, monsters must be attracted to you!{nl}That's it! It was created that way!{nl}
+QUEST_20151022_002833	Working under someone who neither understand or can hear!{nl}Did the person who destroyed our tombstones saw it as a corruption?
+QUEST_20151022_002834	Do you finally understand?{nl}What your utter stupidity has caused?{nl}
+QUEST_20151022_002835	If you understand, start trying to think of a way to revert this.{nl}This simply cannot be ignored.
+QUEST_20151022_002836	I have a request for an investigation{nl}It's the person disguised as Anastospa and destroying Nestospa tombstones.{nl}
+QUEST_20151022_002837	The name is Alvydas. He was at Liugas Access Road.{nl}It might be dangerous, take this flare. Be careful.
+QUEST_20151022_002838	A healing potion needed for a dog.
+QUEST_20151022_002839	That's a healing potion. I will feed it to the dog when you hand it over.
+QUEST_20151022_002840	I will head to where Rose is residing.
+QUEST_20151022_002841	Although it may look simple at first glance, the path of the Swordsman carry some profound meanings.
+QUEST_20151022_002842	I wish there will be more swordsmen capable of helping the villagers.
+QUEST_20151022_002843	Offense is the best defense{nl}There is nothing to fear if you can blow away enemies like the fallen leaves with your two-handed sword.
+QUEST_20151022_002844	There are many brilliant people among the masters that did not let their feminine body affect their training.{nl}If I train diligently, I would be able to catch up to them some time.
+QUEST_20151022_002845	I welcome any swordsman with an aptitude.{nl}Let's defeat the demons together!
+QUEST_20151022_002846	You could say it were the demons that made it possible for us to reach a sub-master level at such a young age.{nl}I will repay them with my sword and shield.
+QUEST_20151022_002847	It's already been 4 years, but there are many people who still carry the scars of that day.{nl}In times like these, our cleric's blessing by the goddess must be shared.
+QUEST_20151022_002848	Huh, are you on a trip?{nl}Must be nice~ I can't move an inch with a mission here.
+QUEST_20151022_002849	To spread the blessing of the goddess, the it doesn't matter what kind of priest you are.{nl}
+QUEST_20151022_002850	Originally, I was following a different path, but I ended up entering the Krivis.{nl}I believe helping others through a conventional way seems most appropriate.
+QUEST_20151022_002851	For tired and sick people, we must become their shoulder rest.{nl}As such, priests must bear with heavy responsibilites.
+QUEST_20151022_002852	For the goddesses in absence, we must be the one to present everyone's hearts.
+QUEST_20151022_002853	Are you asking me to bless the path of the Priest?	Course, I'll gladily issue the blessing.{nl}I pray that your path will be brimming with the blessing of the goddess. 
```

Comment:
There's a lot of spaces after 'Priest?' and before 'Course'

'Course, I'll gladily issue the blessing' - Sounds a bit weird. Issue a blessing? What about: Of course, I would be happy to bless you.